FindBLIS.cmake

```diff
@@ -9,6 +9,8 @@ else()
   set( BLIS_LIBRARY_NAME "blis" )
 endif()
 
+set( BLIS_PREFIX ${BLISROOT} $ENV{BLISROOT} )
```

Comment:
This should be guarded to avoid overriding user input
```
if( NOT DEFINED BLIS_PREFIX )
  set( BLIS_PREFIX ${BLISROOT} $ENV{BLISROOT} )
endif()
```

Reply:
Fixed!